src/AppBundle/Controller/Report/ReportSubmissionController.php

```diff
@@ -30,12 +32,71 @@ public function getAll(Request $request)
     {
         $this->denyAccessUnlessGranted([EntityDir\User::ROLE_DOCUMENT_MANAGE]);
 
-        $this->setJmsSerialiserGroups(self::$jmsGroups);
+        $offset = $request->get('offset', 0);
+        $limit = $request->get('limit', 15);
+        $status = $request->get('status');
+
+        $qb = $this->getRepository(EntityDir\Report\ReportSubmission::class)->createQueryBuilder('rs')
+            ->leftJoin('rs.report', 'r')
+            ->leftJoin('rs.archivedBy', 'ab')
+            ->leftJoin('rs.createdBy', 'cb')
+            ->leftJoin('r.client', 'c')
+            ->leftJoin('rs.documents', 'd')
+            ->orderBy('rs.id', 'DESC');
```

Comment:
Just for info, this query ideally should be a method inside the repository for reuse by other controllers. Keeps the repository queries close to the database and the controllers nice and light. Allowing easier unit testing of both the controllers and repos. Happy to merge in as is though.